tools/dep_updaters/update-base64.sh

```diff
@@ -1,80 +0,0 @@
-#!/bin/sh
-set -e
-# Shell script to update base64 in the source tree to a specific version
```

Comment:
Can you also update the github workflow?

Reply:
Done.